src/Driver/WpcliDriver.php

```diff
@@ -124,21 +119,23 @@ public function wpcli($command, $subcommand, $raw_arguments = [])
             $config = sprintf('--path=%s --url=%s', escapeshellarg($this->path), escapeshellarg($this->url));
         }
 
-        $cmd_output = [];
-        $exit_code  = 0;
+
         $wpcli_args = sprintf('--no-color --require=%1$s/WpcliLogger.php', dirname(__DIR__));
 
         // Query WP-CLI.
-        exec(
-            "{$this->binary} {$config} {$wpcli_args} {$command} {$subcommand} {$arguments} 2>&1",
-            $cmd_output,
-            $exit_code
-        );
-        $cmd_output = implode(PHP_EOL, $cmd_output);
-
-        if ($cmd_output) {
-            // Any output is bad.
-            throw new UnexpectedValueException("WP-CLI driver query failure: {$cmd_output}");
+        $proc = proc_open("{$this->binary} {$config} {$wpcli_args} {$command} {$subcommand} {$arguments}", [
+            1 => ['pipe','w'],
+            2 => ['pipe','w'],
```

Comment:
Should have a space between array elements when on the same line, e..g 

`['pipe', 'w'],`